apps/wing-console/console/ui/src/layout/status-bar.tsx

```diff
@@ -65,11 +82,36 @@ export const StatusBar = ({
           </span>
         </div>
       </div>
+      {/*center*/}
+      <div className="w-full flex justify-center items-center"></div>
```

Comment:
```suggestion
      <div className="grow"></div>
```

You can use `grow` so it occupies all remaining space.